fe/fe-core/src/main/java/com/starrocks/lake/LakeTable.java

```diff
@@ -189,8 +191,12 @@ public Status createTabletsForRestore(int tabletNum, MaterializedIndex index, Gl
         List<Long> shardIds = null;
         try {
             // Ignore the parameter replicationNum
+            WarehouseManager warehouseManager = GlobalStateMgr.getCurrentState().getWarehouseMgr();
+            Warehouse warehouse = warehouseManager.getBackgroundWarehouse();
```

Comment:
the same question, how to identify this is a background function call path only?